Tests/Functional/Asynchronous/Predicates/Expectations/main.swift

```diff
@@ -0,0 +1,83 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/Asynchronous-Predicates
+// RUN: %{built_tests_dir}/Asynchronous-Predicates > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(Linux) || os(FreeBSD)
+    import XCTest
+    import Foundation
+#else
+    import SwiftXCTest
+    import SwiftFoundation
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'PredicateExpectationsTestCase' started at \d+:\d+:\d+\.\d+
+class PredicateExpectationsTestCase: XCTestCase {
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' passed \(\d+\.\d+ seconds\).
+    func test_immediatelyTruePredicateAndObject_passes() {
+        let predicate = NSPredicate(value: true)
+        let object = NSObject()
+        expectation(forPredicate:predicate, evaluatedWithObject: object)
+        waitForExpectations(withTimeout: 0.1)
+    }
+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyFalsePredicateAndObject_fails' started at \d+:\d+:\d+\.\d+
+// CHECK: .*/Tests/Functional/Asynchronous/Predicates/Expectations/main.swift:34: error: PredicateExpectationsTestCase.test_immediatelyFalsePredicateAndObject_fails : Asynchronous wait failed - Exceeded timeout of 0.1 seconds, with unfulfilled expectations: Expect `<NSPredicate: 0x[0-9A-Fa-f]{1,16}>` for object <NSObject: 0x[0-9A-Fa-f]{1,16}>
```

Comment:
Thanks for using the correct line number, `34`, here. I think verifying these appear on the correct line is a good thing, but I know it's painful to keep these expected line numbers in sync with the actual ones that are output. I'm planning on adding a `[[@LINE+<offset>]]` feature soon (originally implemented in #68)--hang tight! 😄